superset-frontend/cypress-base/cypress/integration/dashboard/drilltodetail.test.ts

```diff
@@ -58,220 +58,243 @@ function closeModal() {
   });
 }
 
+function setTopLevelTab(tabName: string) {
+  cy.get("div#TABS-TOP div[role='tab']").contains(tabName).click();
+}
+
 describe('Drill to detail modal', () => {
   before(() => {
-    cy.visit(ECHARTS_DASHBOARD);
-    ECHARTS_CHARTS.forEach(waitForChartLoad);
+    cy.visit(SUPPORTED_CHARTS_DASHBOARD);
   });
 
   beforeEach(() => {
     cy.preserveLogin();
-    closeModal();
   });
 
-  describe('Modal actions', () => {
-    it('opens the modal from the context menu', () => {
-      openModalFromMenu('big_number_total');
-
-      cy.get("[role='dialog'] .draggable-trigger").should(
-        'contain',
-        'Drill to detail: Number of Girls',
-      );
-    });
-
-    it('refreshes the data', () => {
-      openModalFromMenu('big_number_total');
-      // move to the last page
-      cy.get(".pagination-container [role='navigation'] [role='button']")
-        .eq(7)
-        .click();
-      cy.wait('@samples');
-      // reload
-      cy.get("[aria-label='reload']").click();
-      cy.wait('@samples');
-      // make sure it started back from first page
-      cy.get(".pagination-container [role='navigation'] li.active").should(
-        'contain',
-        '1',
-      );
-    });
+  afterEach(() => {
```

Comment:
We should not rely on the afterEach as the test might never finish. Any clean up work should go in the beforEach

Reply:
fixed